src/Momento.Sdk/Exceptions/AlreadyExistsException.cs

```diff
@@ -5,7 +5,10 @@
 /// </summary>
 public class AlreadyExistsException : SdkException
 {
-    public AlreadyExistsException(string message) : base(message)
+    public AlreadyExistsException(string message) : base(message, MomentoErrorCode.ALREADY_EXISTS_ERROR)
+    {
+    }
+    public AlreadyExistsException(string message, MomentoErrorTransportDetails transportDetails) : base(message, MomentoErrorCode.ALREADY_EXISTS_ERROR, transportDetails)
```

Comment:
I know the name `MomentoErrorTransportDetails` is from my spec doc, but curious how folks feel about it.  Open to shortening it if it seems to verbose.

Reply:
I'm fine with it. I prefer self documenting to terse.